html/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>TransiTracker Login</title>
+    <link rel="stylesheet" href="main.css">
+    <link rel="shortcut icon" type="image/jpg" href="html/ECU_logo.jpg"/>
+  </head>
+  <body>
+    <form class = "formLayout" action="url/route" method="post">
+      <main>
+        <img src = "ECU_logo.jpg">
+        <h1 id="login-header">Login</h1>
+
+        <div id="login-error-msg-holder">
+          <p id="login-error-msg">Invalid username <span id="error-msg-second-line">and/or password</span></p>
+        </div>
+
+        <form id="login-form">
+          <input type="text" name="username" id="username-field" class="login-form-field" placeholder="Username">
+          <input type="password" name="password" id="password-field" class="login-form-field" placeholder="Password">
+          <input type="submit" value="Login" id="login-form-submit">
+        </form>
+
+
+
+      </main>
+    </form>
+
+    <script type="text/javascript">
+      // function validate(){
+      //   var username = document.getElementById('username-field').value;
+      //   var password = document.getElementById('password-field').value;
+      //   if (username == admin && pasword == pass){
+      //     alert('success')
+      //     return False;
+      //   }
+      //   else{
+      //     alert('Wrong Password or Username');
+      //   }
+      //}
+    </script>
+  </body>
+</html>
```

Comment:
nothing in the "html" folder is used in the project. It's only there for referencing the styles created earlier in the semester. I'm not sure why this file was included in the PR.